components/brave_extension/extension/brave_extension/_locales/en_US/messages.json

```diff
@@ -171,10 +171,6 @@
     "message": "Change global shield defaults",
     "description": "Message for the button action linking to the settings page"
   },
-  "addBlockElement": {
-    "message": "Block element via selector",
-    "description": "Message for context menu that adds a filter for the cosmetic filtering"
-  },
```

Comment:
I think `Enter element picker mode` might be a bit vague if the context of the `Block element via selector` button right next to it is removed. I imagine we want the text to describe blocking/hiding somehow.

Reply:
maybe we call it "Select elements to block?"

Reply:
Might be worth another opinion from product or design, but uBO uses just `Block element`, which I think is reasonable

Reply:
fixed